riocli/apply/parse.py

```diff
@@ -179,28 +164,6 @@ def parse_dependencies(
                 self._add_graph_node(key)
                 number_of_objects = number_of_objects + 1
 
-        resource_list = []
```

Comment:
thought(non-blocking): This can be considered a breaking change as the output will differ significantly. This can catch people off-gaurd.